lib/iris/tests/experimental/regrid/test_regrid_area_weighted_rectilinear_src_and_grid.py

```diff
@@ -367,25 +361,11 @@ def test_regrid_to_higher_res(self):
         res = regrid_area_weighted(src, dest)
         self.assertCMLApproxData(res, RESULT_DIR + ('higher.cml',))
 
-    @tests.skip_plot
     def test_hybrid_height(self):
         src = self.realistic_cube
         dest = _resampled_grid(src, 0.7, 0.8)
         res = regrid_area_weighted(src, dest)
         self.assertCMLApproxData(res, RESULT_DIR + ('hybridheight.cml',))
-        # Consider a single slice to allow visual tests of altitudes.
-        src = src[1, 2]
-        res = res[1, 2]
-        qplt.pcolormesh(res)
-        self.check_graphic()
-        plt.contourf(res.coord('grid_longitude').points,
-                     res.coord('grid_latitude').points,
-                     res.coord('altitude').points)
-        self.check_graphic()
-        plt.contourf(res.coord('grid_longitude').points,
-                     res.coord('grid_latitude').points,
-                     res.coord('surface_altitude').points)
-        self.check_graphic()
```

Comment:
@marqh Nice. If we have `assertCML` coverage then there's no need for `check_graphic` coverage as well.